### DISCUSSION TITLE
How to console.log in .vitepress/config.js file?

### DISCUSSION THREAD

**Discussion by Animeshz** [vuejs/vitepress#1433] [Q&A] [answered]
Tried searching in docs but no luck...

I'm doing something in `UserConfig.buildEnd` and I want to see some debug info.

```js
export default {
  // ...
  buildEnd: (cfg) => {
    console.log('Hello World')  // doesn't work!!!
  }
}
```

**Comment by brc-dd** [comment, marked as answer]
It's working fine: 
![image](https://user-images.githubusercontent.com/40380293/193920664-e0adec52-edc4-48a6-a94e-56ab6baf9d6c.png)

**Reply by Animeshz** [reply]
Oh yeah, I was doing something stupid I guess, sorry for the issue :sweat_smile:
